src/styles/model/variables.scss

```diff
@@ -75,3 +75,34 @@ $button-width-wide: 22rem;
 
 // transition
 $transition-basic: all 0.2s ease-in-out;
+
+//max-width for media
+$zero-width: 0px;
+$tablet-width: 768px;
+$tablet-above-width: 769px;
+$laptop-width: 1024px;
+$laptop-above-width: 1025px;
+$pc-width: 1370px;
+
+//tablets
```

Comment:
this numbers are from experience or youve searched for appropriate mobile design on Internet?